### ISSUE TITLE
VIM 8 terminal

### ISSUE THREAD

**Issue by blayz3r** [jalvesaq/vimcmdline#31]
Is it possible to convert this plugin to work with vim 8 like NvimR does?

**Comment by jalvesaq** [comment]
Yes. However, I prefer Neovim's terminal because it has syntax highlight.
If someone writes a pull request for this, I may merge it.

**Comment by jalvesaq** [comment]
Closing this issue since nobody made the pull request in seven years...
